HdProduction.HelpDesk.Infrastructure/EfConfigurations/TicketPriorityConfiguration.cs

```diff
@@ -0,0 +1,17 @@
+using HdProduction.HelpDesk.Domain.Entities;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+
+namespace HdProduction.HelpDesk.Infrastructure.EfConfigurations
+{
+    public class TicketPriorityConfiguration : EntityBaseConfiguration<TicketPriority, int>
+    {
+        protected override void ConfigureNext(EntityTypeBuilder<TicketPriority> builder)
+        {
+            builder.Property(p => p.Name)
+                .IsRequired()
+                .HasMaxLength(32);
```

Comment:
I think we should use general const for such consts

Reply:
moved to entity class as const